src/main/java/openblocks/common/tileentity/TileEntityTank.java

```diff
@@ -329,6 +329,8 @@ private void tryBalanceNeighbors(FluidStack contents) {
 			sum += n.tank.getFluidAmount();
 
 		final int suggestedAmount = sum / (count + 1);
+		if (Math.abs(suggestedAmount - contents.amount) < 5) return; // Don't balance small amounts to reduce server load
+
```

Comment:
Hmm, can you split it into separate PR? I'm not really sure I want to pull it now.

Also, do you have any example where this significantly contributed to server lag? Preferably with profiler data.

I'm asking, since this is not 'optimization' in strict sense - it changes behaviour. With this it's harder to fully drain almost empty tanks, since placing another tank under layer of existing ones will only fully drain one directly on top.

Anyway, tanks are mostly used for long term storage and decoration - they will always be significantly more laggier than, for example, opaque barrels when used as buffers. Also, I'm not sure there is configuration where they update constantly without external interaction (never tested it though). Lastly, normally tanks update only once every 20 ticks.